doc-deploy-changelog/action.yml

```diff
@@ -262,7 +262,7 @@ runs:
         if [ $count -gt 0 ]; then
           echo "Using ${{ env.UPDATE_BRANCH }} to update the CHANGELOG file"
           if [[ ${{ env.RELEASE_BRANCH_EXISTS }} == "false" ]]; then
-            git checkout -b ${{ env.UPDATE_BRANCH }}
+            git checkout -b ${{ env.UPDATE_BRANCH }} ${{ env.MAIN_BRANCH }}
```

Comment:
I think we should keep it as `git checkout -b ${{ env.UPDATE_BRANCH }}` because the temporary branch, `maint/changelog-update-v${{ env.TAG_VERSION }}` is only created if the release branch or main do not exist. Otherwise, if the release branch or main do exist, then UPDATE_BRANCH is set to the release branch or main & the changelog file is updated directly on either of those branches

Reply:
Got it... Check 864c9056c889fd2d72ed459f93b2dc84ff711187